server/sonar-server/src/main/java/org/sonar/ce/queue/CeTask.java

```diff
@@ -102,13 +111,24 @@ public int hashCode() {
   }
 
   public static final class Builder {
+    private String organizationUuid;
     private String uuid;
     private String type;
     private String componentUuid;
     private String componentKey;
     private String componentName;
     private String submitterLogin;
 
+    public Builder setOrganizationUuid(String organizationUuid) {
+      this.organizationUuid = organizationUuid;
+      return this;
+    }
+
+    // FIXME remove this method when organization support is added to the Compute Engine queue
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Take the required action to fix the issue indicated by this comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1134)